Web/Startup.cs

```diff
@@ -69,25 +69,29 @@ public void ConfigureServices(IServiceCollection services)
             //if not set just use sqlite
             String DatabaseType = check.checkType() ?? "sqlite";
 
+            //testing with mssl
+            // String DatabaseType = "mssql";
+
             switch (DatabaseType)
             {
                 case "mssql":
-                    var host = _configuration["DBHOST"] ?? "172.19.0.1";
+                    var host = _configuration["DBHOST"] ?? "localhost";
+                    var port = _configuration["DBPORT"] ?? "1444";
                     var db = _configuration["DBNAME"] ?? "openvoting";
-                    var port = _configuration["DBPORT"] ?? "1433";
                     var username = _configuration["DBUSERNAME"] ?? "sa";
-                    var password = _configuration["DBPASSWORD"] ?? "Sql!Expre55";
+                    var password = _configuration["DBPASSWORD"] ?? "SqlExpress!";
 
-                    string connStr = $"Data Source={host},{port};Integrated Security=False;";
-                    connStr += $"User ID={username};Password={password};Database={db};";
-                    connStr += $"Connect Timeout=30;Encrypt=False;TrustServerCertificate=True;ApplicationIntent=ReadWrite;MultiSubnetFailover=False";
+                    string connStr = $"Server=tcp:{host}, {port};Database={db};UID={username};PWD={password};";
+                    // string connStr = $"Data Source={host},{port};Integrated Security=False;";
+                    // connStr += $"User ID={username};Password={password};Database={db};";
+                    // connStr += $"Connect Timeout=30;Encrypt=False;TrustServerCertificate=True;ApplicationIntent=ReadWrite;MultiSubnetFailover=False";
```

Comment:
is there a good reason for us to have this commented out code in master, or can we remove it?

Should at least add a comment explaining what it is and why it is commented out